src/ferc_xbrl_extractor/taxonomy.py

```diff
@@ -1,19 +1,20 @@
 """XBRL prototype structures."""
-from typing import Dict, ForwardRef, List, Literal
+from __future__ import annotations
```

Comment:
Have you tried running this on Python 3.8 / 3.9? I found that the combination of Pydantic's runtime type checking magic and the `__future__` annotations broke things in some cases.

Reply:
I just tested on python 3.8 and 3.9 and I am getting errors as well. Have you found a solution for this?

Reply:
I think the two options are to either not upgrade the type hints, or to only use the most recent version of Python. We went with only using Python 3.10 on the main pudl repo.

Reply:
Ok, I've done the same as PUDL here. I can definitely change that if we have a compelling reason to support python 3.8/9, but I don't personally see one right now

Reply:
Sounds good. Most of this is much more "application" like, and I think in that case just the most recent version will make our lives easier.